modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -341,14 +365,100 @@ dbms.security.oidc.google.token_params=client_secret=GOCSPX-v4cGkygPJvm3Sjjbc0hv
 dbms.security.oidc.google.config=token_type_principal=id_token;token_type_authentication=id_token
 ----
 
-. Log in with your Google SSO credentials using the email address and get the `admin` role when doing so:
-+
-image::sso-configuration-tutorials/oidc-azure-successful-login.png[title="Entra OIDC successful login"]
-+
-[NOTE]
-====
-The native authentication is disabled to prevent someone from logging in to _alice@neo4j-test.com_ with the set password.
-====
+There are then two ways to create the user in the database (which is required to give the users roles from native authorization):
+
+. Using Auth Providers
+xref:authentication-authorization/auth-providers.adoc[Auth Providers] allow us to specify auth providers at the user-level which is useful in this scenario. This approach allows us to have an `admin` user who can authenticate natively, and can then create less privileged users who may only authenticate using `oidc-google` and will receive the roles granted to them using `native` authorization.
+
+.. Change the configuration to allow `native` authentication (for use only by the `admin` user):
+
+[source, properties]
+----
+dbms.security.authentication_providers=oidc-google, native
+----
+
+.. Set the `dbms.security.require_local_user` configuration setting to `true` in the _neo4j.conf_ file. This will switch to User Auth Providers mode whereby users can only authenticate and authorize if they have a corresponding Auth Provider in the database.
```

Comment:
```suggestion
.. Set the `dbms.security.require_local_user` configuration setting to `true` in the _neo4j.conf_ file.
This will switch to User Auth Providers mode whereby users can only authenticate and authorize if they have a corresponding Auth Provider in the database.
```